app/views/shared/check_answers/_children_involved.html.erb

```diff
@@ -1,16 +1,21 @@
 <section class="print-no-break govuk-!-padding-top-6">
-  <%= check_answer_change_link(
-        name: :involved_children,
-        url: providers_legal_aid_application_has_other_involved_children_path(@legal_aid_application),
-        question: t(".involved-children-heading"),
-        read_only:,
-      ) %>
-
-  <%= govuk_summary_list(card: { title: t(".involved-children-heading") }, actions: false) do |summary_list| %>
-    <% @legal_aid_application.involved_children.each_with_index do |child, i| %>
-      <%= summary_list.with_row(html_attributes: { id: "app-check-your-answers__involved_child_#{i + 1}" }) do |row| %>
-        <%= row.with_key(text: "#{t('.child')} #{i + 1}", classes: "govuk-!-width-one-half") %>
-        <%= row.with_value(text: "#{child.full_name} born on #{child.date_of_birth.strftime('%e %B %Y')}") %>
+  <%= govuk_summary_card(title: t(".involved-children-heading"), html_attributes: { id: "app-check-your-answers__children_involved" }) do |card| %>
+    <% unless read_only %>
+      <% card.with_action do %>
```

Comment:
Do actions like this need a visually hidden text to differentiate between them when using a screen reader, rather than them just being announced as a change link?